Formula/apache-zeppelin.rb

```diff
@@ -0,0 +1,44 @@
+class ApacheZeppelin < Formula
+  desc "Web-based notebook that enables interactive data analytics"
+  homepage "https://zeppelin.apache.org"
+  url "https://www.apache.org/dyn/closer.lua?path=zeppelin/zeppelin-0.6.1/zeppelin-0.6.1-bin-all.tgz"
+  sha256 "5185952361d1999fb3c00b8aca2c6ddc57353af5a1fd93f4f5b9dd89c7222147"
+  head "https://github.com/apache/zeppelin.git"
+
+  bottle :unneeded
+
+  def install
+    rm_f Dir["bin/*.cmd"]
+    libexec.install Dir["*"]
+    bin.write_exec_script Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    begin
+      ENV["ZEPPELIN_LOG_DIR"] = "logs"
+      ENV["ZEPPELIN_PID_DIR"] = "pid"
+      ENV["ZEPPELIN_CONF_DIR"] = "#{testpath}/conf"
+      conf=testpath/"conf"
+      conf.mkdir
+      (conf/"zeppelin-env.sh").write <<-EOF
+      export ZEPPELIN_WAR_TEMPDIR="#{testpath}/webapps"
+      export ZEPPELIN_PORT=9999
+      EOF
```

Comment:
It seems that I should be able to use `~EOF` instead of `-EOF` to remove leading space, however I get the following error when I try : 
```
/usr/local/Library/Homebrew/extend/pathname.rb:139:in `write'
ArgumentError: wrong number of arguments (0 for 1+)
```
Any idea ?